base/doc/ltnews33.tex

```diff
@@ -358,23 +512,20 @@ \subsection{Execute \cs{par} at the end of \cs{marginpar} arguments}
 \githubissue{489}
 
 
+\subsection{Delay change of font series and shape to \cs{selectfont} call}
 
-\subsection{Producing several footnote marks to one footnote}
-
-It is sometimes necessary to reference the same footnote several
-times, i.e., produce several footnote marks with the same number or
-symbol. This is now always possible by placing a \cs{label} into the
-\cs{footnote} and reference it with the command \cs{footref}
-elsewhere.  This way marks refering to footnotes anywhere on the page
-(including those in \texttt{minipage}s) can be generated.  In the past
-this command was only available with certain classes or when loading
-the \pkg{footmisc} package.
+With the NFSS extensions introduced in 2020 the font series and shape
+settings be be influenced by changes to the font family. The setting
```

Comment:
"be" doubled

Reply:
five others elsewhere as well, thanks ... I should always use Nelson's "dw" command for checking :-(